File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatablelistfragment/DataTableListMvpView.java
package com.mifos.mifosxdroid.online.datatablelistfragment;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.client.Client;

public interface DataTableListMvpView extends MvpView {

    void showMessage(int messageId);

    void showMessage(String message);

    void showClientCreatedSuccessfully(Client client);

    void showWaitingForCheckerApproval(int message);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatablelistfragment/DataTableListMvpView.kt
package com.mifos.mifosxdroid.online.datatablelistfragment

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.client.Client

interface DataTableListMvpView : MvpView {
    fun showMessage(messageId: Int)
    fun showMessage(message: String?)
    fun showClientCreatedSuccessfully(client: Client)
    fun showWaitingForCheckerApproval(message: Int)
}